config/prod/mpnst-analysis.yaml

```diff
@@ -0,0 +1,23 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2.yaml
+stack_name: mpnst-analysis
+dependencies:
+  - peer-vpn-computevpc
+parameters:
+  # The Sage deparment for this resource
+  Department: "Computational Oncology"
+  # The Sage project this resource will be used for
+  Project: "NTAP"
+  # The resource owner
+  OwnerEmail: "xindi.guo@sagebase.org"
+  AMIId: "ami-0c031218cbaf1204f"
```

Comment:
is this AMI derived from an AMAZON linux instance?

Reply:
I believe so. @thomasyu888 ?